content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,163 @@
+
+You'll learn how to provision a Prisma Postgres database on your workspace as a partner, and how to transfer it to another user's workspace so they can "claim" the database. We'll cover how the process is secured using OAuth2, and by the end, you'll understand the full flow and how to integrate it into your own product experience.
+
+This guide references the actual implementation in the `npx create-db` CLI and Cloudflare Workers as real world examples. The repo for the `npx create-db` is [here](https://github.com/prisma/create-db), which can be used as a reference for how to use the Partner API in your own projects.
+
+:::note How does this fit into your app?
+The two workers in this guide are just reference examples. You would typically build this logic into your own backend or serverless functions.
+
+Similarly, the `npx create-db` CLI is a simple demo. In your product, you can trigger the same API calls from your own UI or onboarding flows to create a seamless experience for your users.
+:::
+
+## Core Concepts
+
+Before diving into implementation, let's clarify the main concepts involved in the Partner API integration:
+
+- **Partner API**: A set of endpoints that allow approved partners to provision and manage Prisma Postgres databases on behalf of their users.
+- **Projects & Databases**: In Prisma Postgres, a "project" represents a database instance. When you provision a project, you are creating a new database for your user.
+- **Authentication**: All API requests require authentication. As a partner, you use OAuth2 to obtain integration tokens (for your app) and facilitate secure transfers to your users.
+- **Tokens**: There are two main types of tokens:
+  - **Integration Token**: Issued to your partner integration, used to provision and manage databases.
+  - **User Access Token**: Obtained via OAuth2 when a user authenticates, used to transfer database ownership to the user's workspace.
+
+## How to become a partner
+
+To use the Prisma Postgres Partner API, you first need to set up as a partner:
+
+1. **Request access to the Partner API**: Contact the Prisma team from the [Prisma Partners page](https://www.prisma.io/partners) to request access to the Partner API. You will be guided through the onboarding process.
+2. **Obtain OAuth credentials**: Once approved, you will receive an OAuth client ID and client secret. These credentials are required to authenticate your integration and enable secure database transfers for your users.
+
+For a complete list of available endpoints and details on request/response formats, see the [Prisma Management API documentation](/postgres/database/api-reference/management-api).
+
+## Provisioning a Database as a Partner
+
+To provision a new Prisma Postgres database for your users as a partner, follow these steps:
+
+1. **Collect user input**: Gather the necessary information from your user, such as the desired region, database name, and any other options your app supports.
```

Comment:
Let's make this less specific, you wouldn't always need to collect e.g. region, etc from the user, your application logic may simply be able to prepare that.
This step is rather to gather the info (as you also mentioned here), which may or may not come from user input.